doc/modules/linear_model.rst

```diff
@@ -1050,6 +1059,67 @@ considering only a random subset of all possible combinations.
 
     .. [#f2] T. Kärkkäinen and S. Äyrämö: `On Computation of Spatial Median for Robust Data Mining. <http://users.jyu.fi/~samiayr/pdf/ayramo_eurogen05.pdf>`_
 
+.. _huber_regression:
+
+Huber Regression
+----------------
+
+The :class:`HuberRegressor` is similar to :class:`Ridge` that it also applies
+L2 regularization and a squared loss for samples which are classified as inliers.
+A sample is classified as an inlier if the absolute error of that sample is
+lesser than a certain threshold. It differs from :class:`TheilSenRegressor`
```

Comment:
This reads weirdly. "It's similar to Ridge because X" "It's different from TheilSen because Y". I would first expect to hear what the difference with Ridge is!